prow/bump.sh

```diff
@@ -82,13 +81,13 @@ if [[ "$cmd" == "--push" ]]; then
   fi
   echo -e "Pushing $(color-version ${new_version}) via $(color-target //prow:release-push --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64) ..." >&2
   bazel run //prow:release-push --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64
-elif [[ "$cmd" == "--list" ]]; then
+elif [[ -z "$cmd" || "$cmd" == "--list" ]]; then
   # TODO(fejta): figure out why the timestamp on all these image is 1969...
   # Then we'll be able to just sort them.
   echo "Listing recent versions..." >&2
   options=(
-    $(gcloud container images list-tags gcr.io/k8s-prow/plank --filter="tags ~ ^v|,v" --format='value(tags)' \
-      | grep -o -E 'v[0-9]{8}-[0-9a-f]{6,9}' | sort -u | tail -n 10)
+    $(gcloud container images list-tags gcr.io/k8s-prow/plank --limit=10 --format='value(tags)' \
+      | grep -o -E 'v[^,]+' | tac)
```

Comment:
mind=blown. I didn't know `tac` was a thing. amazing
/meow space

Reply:
`gcloud container images list-tags` is already sorted in reverse order, with the most recent tag first. isn't this doing the wrong thing?